### PR TITLE
Remove suffix from circos label

### PR DESCRIPTION
**Pull request by joko3ono**
Don't divide the numbers by 1000. Just keep the real lengths. (rather than dividing by 1000, and adding "k").
Don't show the units. So instead of "230 bp", just "230")

### CODE REVIEW

**Review by tadast — APPROVED** [review]
Code LGTM, let's get Daniel or Yannick to review the labels are not too hard to parse to humans now :)

**Comment by djleite** [comment]
@tadast can this be deployed to staging to view/test it?

**Comment by djleite** [comment]
@joko3ono @tadast looks fine to me, not too cramped or too many labels. I tested for seqs between 100-5,000,000 bp

**Comment by tadast** [comment]
@joko3ono could you please rebase? Do you not have access to this repo to create branches? If you do, please work on this repo instead of your fork, so that I could rebase before merging if needed 🙏

**Comment by joko3ono** [comment]
@tadast rebased, ready to be merged
